src/modality-specific-files/task-events.md

```diff
@@ -98,12 +99,42 @@ In the accompanying JSON sidecar, the `trial_type` column might look as follows:
             "start": "A red square is displayed to indicate starting",
             "stop": "A blue square is displayed to indicate stopping"
         }
+    },
+    "channel": {
+        "Description": "Channel(s) associated with the event",
+        "Delimiter": "|"
+    },
+    "artifact": {
+        "Description": "Description of an artifact."
     }
 }
 ```
 
-Note that all other columns SHOULD also be described but are omitted for the
-sake of brevity.
+Note that in the example above:
+
+1.  Only a subset of columns are described for the sake of brevity.
+    In a real dataset, all other columns SHOULD also be described.
+
+1.  The `channel` column contains a list of values that are separated
```

Comment:
This is confusing.  I am liking the idea of the `Delimiter` field being optional less and less. 

I somehow thought from the discussion that we were going with  the `channel` column is ONLY interpreted as a list if the `Delimiter` is given in the JSON sidecar?  Also, the next line sounds like we have more than one default delimiter.  If there is a default delimiter -- it should only be the comma?